orbitrap_training.py

```diff
@@ -19,16 +19,15 @@
 orbitrap_regexp_terms = re.compile(".*orbitrap.*|.*hcd.*|.*q-exactive.*|.*lumos.*|.*velos.*", re.IGNORECASE)
```

Comment:
This script can be a bit cleaner/clearer. 
You can use more functions that can be generalized and therefore reused for the other models as well. So for instance a function for training a model. 

After creating these generalized functions, I think it is best to create a jupyter notebook for selecting and training. You can in that case make 1 notebook for all 4 specific models and still keep it clear, by adding clear markdown. Using 4 scripts that have the code behind if __name__ == "__main__":  is also a fine alternative.

Reply:
Yes! I initially wanted to make a more general function for training but decided to go with these scripts first. I will try making a more general function later on